runtime/lua/vim/filetype.lua

```diff
@@ -1232,27 +1204,20 @@ local extension = {
   sgm = function(path, bufnr)
     return require('vim.filetype.detect').sgml(bufnr)
   end,
-  rc = function(path, bufnr)
-    if not path:find('/etc/Muttrc%.d/') then
-      return 'rc'
-    end
-  end,
-  rch = function(path, bufnr)
-    if not path:find('/etc/Muttrc%.d/') then
-      return 'rc'
+  t = function(path, bufnr)
+    if not require('vim.filetype.detect').nroff(bufnr) and not require('vim.filetype.detect').perl(path, bufnr) then
+      return 'tads'
     end
   end,
-  control = function(path, bufnr)
-    return require('vim.filetype.detect').control(bufnr)
-  end,
-  copyright = function(path, bufnr)
-    return require('vim.filetype.detect').copyright(bufnr)
-  end,
   -- Ignored extensions
   bak = function(path, bufnr)
     local root = vim.fn.fnamemodify(path, ':r')
```

Comment:
would it make sense to extend `vim.fs.basename` to cover this? (@gpanders)

Reply:
We could add `rootname` maybe? Though I don't think we need to add a `vim.fs` function for each fnamemodify variant. Personally I'm fine with sticking with fnamemodify here.